newrelic.ini

```diff
@@ -52,7 +52,7 @@ monitor_mode = true
 # write out a log file, it is also possible to say "stderr" and
 # output to standard error output. This would normally result in
 # output appearing in your web server log.
-#log_file = /tmp/newrelic-python-agent.log
+#log_file = stdout
```

Comment:
Is this change intended?

Reply:
Yes. I simply wanted a reminder that if we use the `log_file` for debugging -- like I did --- we can/should use `stdout` instead of a log file on the pod itself, because it's much easier to look at the log data either in GCP or via `kubectl logs -f <pod-name>`.